indico/MaKaC/webinterface/rh/authorDisplay.py

```diff
@@ -14,25 +14,38 @@
 # You should have received a copy of the GNU General Public License
 # along with Indico; if not, see <http://www.gnu.org/licenses/>.
 
+from MaKaC.webinterface.displayMgr import ConfDisplayMgrRegistery
+
 from MaKaC.webinterface.pages import authors
 from MaKaC.webinterface import urlHandlers
 from MaKaC.webinterface.rh.contribDisplay import RHContributionDisplayBase
-from MaKaC.errors import MaKaCError
+from MaKaC.errors import MaKaCError, AccessError
+
+
+class RHAuthorDisplayBase(RHContributionDisplayBase):
 
-class RHAuthorDisplayBase( RHContributionDisplayBase ):
+    _MENU_ITEMS_TO_DISABLE = ('speakerIndex', 'authorIndex', 'contributionList')
```

Comment:
> For the same reasons the Author Details page
>
> https://indico-sandbox.cern.ch/event/18/contribution/0/author/0
>
> should be hidden if the other lists are disabled. Or this should be made otherwise configurable.

I thought that if any of these items is disabled then the access to the author page should be disabled.